templates/rhdm71-prod-immutable-kieserver.yaml

```diff
@@ -387,22 +506,22 @@ objects:
             value: "${APPLICATION_NAME}-kieserver"
           - name: KIE_SERVER_USE_SECURE_ROUTE_NAME
             value: "${KIE_SERVER_USE_SECURE_ROUTE_NAME}"
-          - name: KIE_SERVER_PWD
-            value: "${KIE_SERVER_PWD}"
           - name: KIE_SERVER_USER
             value: "${KIE_SERVER_USER}"
+          - name: KIE_SERVER_PWD
+            value: "${KIE_SERVER_PWD}"
           - name: KIE_SERVER_CONTAINER_DEPLOYMENT
             value: "${KIE_SERVER_CONTAINER_DEPLOYMENT}"
           - name: MAVEN_REPOS
-            value: "RHDMCENTR,EXTERNAL"
-          - name: RHDMCENTR_MAVEN_REPO_SERVICE
-            value: "${DECISION_CENTRAL_MAVEN_SERVICE}"
-          - name: RHDMCENTR_MAVEN_REPO_PATH
-            value: "/maven2/"
-          - name: RHDMCENTR_MAVEN_REPO_USERNAME
-            value: "${DECISION_CENTRAL_MAVEN_USERNAME}"
-          - name: RHDMCENTR_MAVEN_REPO_PASSWORD
-            value: "${DECISION_CENTRAL_MAVEN_PASSWORD}"
+            value: "INTERNAL,EXTERNAL"
+          - name: INTERNAL_MAVEN_REPO_SERVICE
+            value: "${INTERNAL_MAVEN_SERVICE}"
```

Comment:
Why was DECISION_CENTRAL_MAVEN_SERVICE renamed to INTERNAL_MAVEN_SERVICE?

Reply:
because this template does not include decision central, IMHO it makes more sense.

Reply:
I'm going to change it back to DECISION_CENTRAL_MAVEN_SERVICE. All our other template examples in RHPAM allow you to specify the BC maven service + an external maven URL. If you don't have decision central deployed, that's cool, you just use the external maven URL.